_i18n/en.yml

```diff
@@ -786,6 +786,9 @@ library:
   moneroaddressescheatsheet20201206: "Monero Addresses Cheatsheet (version 20201206)"
   moneroaddressescheatsheet20201206p: >
     Between the ‘big picture’ and the mathematical details, a single-page recap of various keys, addresses, scopes (private/public, spend/view, on-chain/off-chain, payer’s/payee’s) and their relations.
+  ringscheatsheet20210301: "Rings Cheatsheet (version 20210301)"
```

Comment:
I would not put the version. Just for readability, not a hard request

Reply:
yep I agree, at least until next version will appear (if ever ;) )... so I would delete version from Addresses Cheatsheet as well...
By the way, in your opinion, what about making a "Cheatsheets" area between "Books" and "Magazines"? If you like it and it was my duty as proponent, can you provide some hints? (I'm worried to make a mess acting on site structure)

Reply:
> until next version will appear

I'm not sure would be necessary even in that case. You put the version number inside the PDFs, i would say that's enough.

> I would delete version from Addresses Cheatsheet as well

yes, i'm taking care of that right now.

> what about making a "Cheatsheets" area between "Books" and "Magazines"?

Yes that's a good idea. I'll take care of that too after this PR is merged.